examples/customresources/demos/complexmodules/moduletest/module_test.go

```diff
@@ -0,0 +1,171 @@
+//go:build !no_tflite
```

Comment:
Why guard on tflite?

Reply:
not sure, but the test file was copied from complexmodule and seems to have been last changed for the 32bit build